### DISCUSSION TITLE
Problem with flex grow not being applied when container query is used

### DISCUSSION THREAD

**Discussion by llighter** [tailwindlabs/tailwindcss#10627] [Help] [answered]
When placing multiple card UIs on the screen, **flex grow** is used to adjust the height to the longest data. However, when using flex grow, **if the container query is applied**, flex grow does not seem to be applied, so I wonder if there is a solution.

sample)
<img width="829" alt="image" src="https://user-images.githubusercontent.com/11993431/219943866-30646093-7ebb-4085-9b73-df3319031a3a.png">

The first card in the capture is the long item and the second is the height adjusted using flex grow. And the last card is the second card wrapped in container query.

You can see that the height of the third card does not match.

I put a sample example on the tailwind playground and you can see the code in action right away.
🔗 https://play.tailwindcss.com/MJW0gxe5Tr?size=1104x720

**Comment by wongjn** [comment, marked as answer]
The unexpected behavior derives from the fact that the extra `<div>` element means the height is no longer "inherited" down through to the card flex layout – the container query has no bearing on this. [Here is the example fixed](https://play.tailwindcss.com/RsVftTMR1b?size=1104x720) by applying `height: 100%` via `h-full` to the child element.

**Reply by llighter** [reply]
The height was being determined based on the div added to use the container query. When I apply h-full to the child div, it works perfectly.

Thank you so much.

| div(container query)  | child div(flex without h-full) | child div(flex with h-full) |
| ------------- | ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/11993431/219949110-d3e32dfd-055f-4c05-a38f-35d764c66983.png)  |  ![image](https://user-images.githubusercontent.com/11993431/219949320-c25dd69f-9898-41ee-b1be-006c28bae5f6.png)   | ![image](https://user-images.githubusercontent.com/11993431/219949477-05e3a70c-0592-4370-bc3e-b18465977294.png)  |
